Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -443,86 +443,110 @@
-workflows[140.078] = ['',['RunMuonEG2022E','HLTDR3_2023','RECONANORUN3_reHLT','HARVESTRUN3']]
+workflows[140.065505] = ['Run3-2023_JetMET2022D_RecoPixelOnlyTripletsCPU',['RunJetMET2022D','HLTDR3_2022','RECODR3_reHLT_Patatrack_PixelOnlyTripletsCPU','HARVESTRUN3_pixelTrackingOnly']]
+workflows[140.065511] = ['Run3-2023_JetMET2022D_RecoECALOnlyCPU',['RunJetMET2022D','HLTDR3_2022','RECODR3_reHLT_ECALOnlyCPU','HARVESTRUN3_ECALOnly']]
+workflows[140.065521] = ['Run3-2023_JetMET2022D_RecoHCALOnlyCPU',['RunJetMET2022D','HLTDR3_2022','RECODR3_reHLT_HCALOnlyCPU','HARVESTRUN3_HCALOnly']]
+
+workflows[140.071] = ['',['RunMuon2022E','HLTDR3_2022','RECONANORUN3_reHLT','HARVESTRUN3']]
+workflows[140.072] = ['',['RunZeroBias2022E','HLTDR3_2022','RECONANORUN3_ZB_reHLT','HARVESTRUN3_ZB']]
+workflows[140.073] = ['',['RunBTagMu2022E','HLTDR3_2022','RECONANORUN3_reHLT','HARVESTRUN3']]
+workflows[140.074] = ['',['RunJetMET2022E','HLTDR3_2022','RECONANORUN3_reHLT','HARVESTRUN3']]
+workflows[140.075] = ['',['RunDisplacedJet2022E','HLTDR3_2022','RECONANORUN3_reHLT','HARVESTRUN3']]
+workflows[140.076] = ['',['RunEGamma2022E','HLTDR3_2022','RECONANORUN3_reHLT','HARVESTRUN3']]
+workflows[140.077] = ['',['RunTau2022E','HLTDR3_2022','RECONANORUN3_reHLT','HARVESTRUN3']]
+workflows[140.078] = ['',['RunMuonEG2022E','HLTDR3_2022','RECONANORUN3_reHLT','HARVESTRUN3']]
 
 ### run3 (2023) skims ###
-workflows[140.104] = ['',['RunZeroBias2022D','HLTDR3_2023','RECONANORUN3_ZB_reHLT','SKIMZEROBIASRUN3_reHLT']]
-workflows[140.105] = ['',['RunBTagMu2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMBTAGMURUN3_reHLT']]
-workflows[140.106] = ['',['RunJetMET2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMJETMET0RUN3_reHLT']]
-workflows[140.107] = ['',['RunDisplacedJet2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMDISPLACEDJETRUN3_reHLT']]
-workflows[140.109] = ['',['RunEGamma2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMEGAMMA0RUN3_reHLT']]
-workflows[140.110] = ['',['RunTau2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMTAURUN3_reHLT']]
-workflows[140.111] = ['',['RunMuon2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMMUON0RUN3_reHLT']]
-workflows[140.112] = ['',['RunMuonEG2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMMUONEGRUN3_reHLT']]
-workflows[140.113] = ['',['RunNoBPTX2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMNOBPTXRUN3_reHLT']]
-workflows[140.114] = ['',['RunHcalNZS2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMHCALNZSRUN3_reHLT']]
-workflows[140.115] = ['',['RunHLTPhysics2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMHLTPHYSICSRUN3_reHLT']]
-workflows[140.116] = ['',['RunCommissioning2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMCOMMISSIONINGRUN3_reHLT']]
-workflows[140.117] = ['',['RunCosmics2022D','HLTDR3_2023','RECOCOSMRUN3_reHLT','SKIMCOSMICSRUN3_reHLT']]
-#workflows[140.118] = ['',['RunParkingBPH2022B','HLTDR3_2023','RECONANORUN3_reHLT','SKIMPARKINGBPHRUN3_reHLT']]
+workflows[140.104] = ['',['RunZeroBias2022D','HLTDR3_2022','RECONANORUN3_reHLT','SKIMZEROBIASRUN3_reHLT']]
```

Comment:
we need a ZB flavor

Reply:
> we need a ZB flavor

ah, there is  `140.063] = ['',['RunZeroBias2022D'`.
Is this just a no-DQM production like setup?

Reply:
Good question @slava77 

Looking in [master](https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_standard.py#L497C1-L497C1), this workflow produce DQM from [RECONANORUN3_ZB_reHLT_2022](https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_steps.py#L2910) (or RECONANORUN3_reHLT in 13_0). But there is no harvesting step.

@cms-sw/pdmv-l2 Is this your intention, or missing?

Reply:
> master

@srimanob @slava77 I am not sure of the use case of WFs 140.104-140.118. For consistency, I changed the DQM sequence to ZeroBias 140.104 as well but missed the changes in the Harvesting step. As the ZeroBias DQM request is initially from tracking @mmusich, as per their need either we can revert back to standard DQM or update harvesting step for this WF.